tests/unit/test_instance.py

```diff
@@ -291,6 +298,49 @@ def test_create_with_processing_units(self):
         self.assertEqual(instance.labels, self.LABELS)
         self.assertEqual(metadata, [("google-cloud-resource-prefix", instance.name)])
 
+    def test_create_with_autoscaling_config(self):
```

Comment:
Update any existing test eg: `test_create_with_processing_units` and add `autoscaling_config` with a comment that `autoscaling_config` should always be set irrespective of `processing_unit` or `node_count`